src/wasm-features.h

```diff
@@ -102,6 +108,9 @@ struct FeatureSet {
     if (hasSIMD()) {
       f(SIMD);
     }
+    if (hasExceptionHandling()) {
```

Comment:
It's hard to tell (and is not commented!), but the features are iterated over in alphabetical order according to their LLVM name, so ExceptionHandling should be up after BulkMemory. This keeps the target features section in alphabetical order, which is nice.

Reply:
Done